package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vtex/delivery-packages",
-  "version": "2.17.0",
+  "version": "2.17.1-beta",
```

Comment:
Just generate a stable tag and erase the beta commit :D (We generally create a beta branch to deal with these tests)

Reply:
Will do!